step-api/src/test/java/org/jenkinsci/plugins/workflow/steps/StepConfigTester.java

```diff
@@ -30,6 +30,7 @@
 import hudson.tasks.BuildStepDescriptor;
 import hudson.tasks.Builder;
 import static org.junit.Assert.*;
+
```

Comment:
@jtnord if you are going to be nit picky on people using `l` as a variable identifier and the perfectly cromulent word iff (which can only embiggen your soul) *and* you insist on using a brain-dead IDE, please do not submit pull requests with gratuitous whitespace changes ;-)